marklogic-data-hub/src/server-side/impl/trace-lib.xqy

```diff
@@ -610,7 +610,7 @@ declare function trace:get-traces($page as xs:int, $page-length as xs:int)
   let $end := $start + $page-length - 1
   let $count := xdmp:estimate(/trace)
   let $traces :=
-    for $trace in cts:search(/trace, cts:true-query(), ("unfiltered", cts:index-order(cts:element-reference(xs:QName("created")), "descending")))[$start to $end]
+    for $trace in cts:search(/trace, cts:true-query(), ("unfiltered", cts:index-order(cts:path-reference("/trace/created"), "descending")))[$start to $end]
```

Comment:
has the traceDB config been updated to add this path range index?

Reply:
it never had a range index. dunno how it ever worked.